assets/js/domain-verifier-checker.js

```diff
@@ -0,0 +1,164 @@
+//Browserify was used to create domain-verifier-bundle.js so that we can use 'require' in the browser 
```

Comment:
Consider running this file through something like Prettier so that the formatting is consistent. Currently, a lot of different styles are mixed together (semicolons, no semicolons, inconsistent spacing, etc.) [Example - could probably just copy this from the Prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6VAhAThA7gZziwEsAzATwAJcBDfSgV0IBNKYJKws4b5LmIAWxrEoAWgBuRMsSJiARgyjMANnAB0AK3r4OMABa9qcTjSiNClAOTcAjg2LcrlUW30n5OAkUoAdKP6QUPgwnBDMcGCUALyUdg7cABQ2xAAOqWoKojRY5GKQEWBWAJQA3IHQIZQ0zMzc+PgAwuGRMXFw9o5wySTpmTV1cA35LUVlFcGhANZw5M2FbfFdPWkZcGIz5KkiWPgl5QFQQVUAKgCSAAoAym1WADypAHyN7mBTLqRuJqTEai70NDAMAYNBUKio+ggITgrCMBhMAFUAEoAGWq8ggUgANJxXu9yBAGFhKIQsFIsL5fFZ6AAJE4na4kuAwGCiADmAOUnCIrJ+YF4cBxZlYwhmJKJJgJRKZZJ8qRwEmIEXo8PanW4rDuNEo+m4pGivhAAA8sKkxGpmGy5OxBCp1PoYLaAMSQXZiQjA1KG54AeSRN3cNSI6juqBoj1DTysE1OPoAsiiAPoXACCJxpbUNqHUuDgYI2UDwUFQJrNFqtWHUNpUhoOx1CjRRKauV0TAHEfT6ACKZkDyGpWyj9y3rfAMMBgIb4WsxhtNluJjApnuxQ3Dwfr9bMMwVmeHdAnfTEeikJRA4jQSiioZfSgqY8wFWGUKEmCpBihVUhXhMdT+U9HKyl7XiiEBsgAorAuSJPARowDiWA0LgxSUMA-iUBhd7MneYHoZhZCUIkiHIaheGYThbJtAAJMkdz3hGqD0SU6g0OkCDMCcEDJE6KhgcxrGpOxMFwHBxRkZQAC+lB5pYaHmORFHUbR9GhkxxRViJMDCXBADUhp+CA6kCexnHcbxbIlOJEnieZ6j1BAKhSG0AHntA2kwChckKe0wJYOYNH3Pg2xQAxQVmI8zEOra7lGWxyimeZYnyRh1nJT5RLmOZ-ipegADM6h+MWqAAGKiKwqoSKCSq8BAxIIDAuQuOYqonPGaI-H8wqUOSZBUF+xBslAP7cJQECfKqvDwN+QFQH+Q34OQRyUC5M2UNsuxwImpYqIm1aJNuMA0Di77yPeYCJpsibuEax0MKdxDnZsOKCFONBWp5eFqKE60sJ92HmQAjG0IFgZBDXkIkhoXDk+DspQrUJvwvA0OoqOGklGHg6RaU-TCbQIyi+3IxjmGA3ZQwOVIkMgFcCKNI04EtujLG1I0Kh0PgiSNs2rYdt2JNSfyMBgPoiRwB9aVk-ZjndOLLPMGzHNc-OrZLl2JMYdwvmHCl-h4ccDkaOZiS48wSV-aElX3gdtWJvVJA3rEpvqAAaimKJnF2aZ+lcBx4QRiRW9V7BYHbUGyPgEvkQHACEKZYIh5DqMe8eJ4HVU26H9sR8UUcKSDEFQRDhouxnNW7EgzPS1ThoAOo4FAlEwFsJhiCSkIMCorDyCYh2nesOSJ8zAyKw0ys84uy4a5J0kqLJ4kYV93VlyHiakISXKxKQoKEAvy21YRxBtAADJQpQuJQdzL9b5dhw1EfqGojcGOfxA6TpxRyXvi-Yak7yxEHTOd8Hb4AANrEAALqgKsCdM6F1ZhWAgXrNK0dSAm3-k7O6cDNif2Qd5cigDb7ryUKwWIDUGBwG-qTbCk0hiHVWgAletts5DHAVAqwtDpoXigIgvB+DMa5C8vwzClViQ9QoG0TY8xIjqHERDOgYBiDEF2hAK6IkaCJBeg0N64ssScPodwrEsCHrwPIGoo009vKpWEaYYW+hCLiyETYguYNoKGi7EIEQ5gXbSD5LwbhlBioiAtFXCmMsxZGVZuzMe3MFxq0sQpVKVCMIBzkbglB+CXFF2ph44QrgfEkD8atK445JwwhhMPKJStYm807OrZJkkGkyTgE44RWTwY5M8fk3xD1-GXiCb8Cphl5aj05jUye9SMnkWsfgmZ0y96x0IavYhyh0n8PaW4kAh4TBLIPpsfexIDDHivGYMgdDqB0EoIWUIKzWCuBam1ZagzKkK2iWMlWEyElzMafJGZzTWkbOLiAAAchwUuN8Q70GKhvZgoTdDhMNAMlQEoXmjPHnEqeeFUpJKKgAJjmugLskQhCpChL3dwJyoBnKqOwUa8hDr3IpQIPJ5gurGKiJsAlqAESWHhLyjgXAeB8FVFo-AOi3BGHwB3LuOoaBOR7ggEkA0oB43kH1CluysDUjWiQSqfB9kwQpfovpUB1L+HQG2PQegmVdNZVyORt5YaDWGiYMat5jWrR+HmO5zUjXpBwPKYgApr7BwPvbJOhUVoBNxomYQVLSB0MSGszC4l6yUuukpKwTo43UpgMxK2SabLYTjW0AoMjCgtE0WYa608l7stMeY4GZhoEXCwQ9AA0ggpBktsKKFIJ8WIKrcCUAwAwftRATZtserMcxOIrDXWYuwAAUlcH0wKk3qAOtqQqRbQjMq8W0a6KjEwKKUVWqA0DcleMQbW3+U7TFtAGPUJoox1AIDLaCiIrb7pgESH20g09xIHiPPQY5IrXqDgMEYWgOhlWqvVTsphWr6CBr1SYTlu6rwQZMKuEAoD92iDEB67hCheLyEriAHSBGoB6WQIaHS9bNi0aQSAVNlRQhEhUL2B0b58BIHQPpHS-BbWUCEwTZMaYaTILY5MRSsRAXJBeJEKYcNnBCc46J6wqN1CWR1phKiLFNA0CNIkVp5FONIEYFgFQWIqFbpOC3SzVhYJ5ts1M8UE4pyWajW5Lded+G2Wrt0KwxUfQImBV2fiVSYkfL5pMmxMbtoqOilu26P6G3XTS9g2Yz1sMJJSm5-hRAcBYG82eGaiRNC2CNLqHE00mA4mK7Vfz+D8C4GIHYwi9XI7YxsRhfklhnPEBeq+Kwld3PeWK7cc4cZwJhZONGCbClPA8CmA0-rdATAcIxFgPN42+uYSm7EKwKYMB+hOOBSL63MIrZoGtpbG3BtNa1ftg70kE63HAkiJEfpFtvaHNwO712Ijb07jAV7B2jvWHC+24FPpa7Ar+8I753lAthKpsVyJrzqkfPiSxUg8AsCJHONcfLPzplJRxea1AlBgVkGbjqESYZ8CKKPu4FQgkK6FXQDx1IfH0C4EFzmXK9kiSTnUJAQQqAjOVRZ70GAYgRJEEUYQfA0u5UKPl+6e+jdFdGmV8edYuKAAc6hUj6C9FAHz5gj3sBPSzs9IQsAAySl-GAojGdGgwrEJ3AMqwQCuDrtkSa-bu5yCSBqtwrCh-XsSdOxJzCxGPufcwV9rqPwQGyF+VzROxFxRLXwMBC9O9z5QQPJBG7qFIDgQQLwcjSJNjDOAZxYCJHT2OeQTvEhQBxPnnEAMABsudQ9JIasyDKEesAHBxTzlkfP+OoEF7gYXousDi8l+r2XXA0gK6V2vw3auZea539rivbI9cG8IGIXFAB2M3Fv-zlYCae5RdvroaKd67vWYfiSDyBrEUBCBGPA+ePHPJPHELjH3BqDPZ+fQFPS+O8eA9+FCBeN3QvD3DNWIYFBgQQHuInJ3CXQwLAaRFMLSU1dSdgcvdkRIQfcYAIIvQ6BOP3d8KVNvESOgwvH5QvLWCfP-LQCAUQZIfYbKZBdAP3bnVAQPHIT8DVHpIWaNHAScBoagdwX1EwJgHweqIgAESlXNOaa3WVZQNQRMDvQQDrMWKQWAFrOASwmAM3bgWwolUHFQLSOgjCcSAKcmMcFwsQVkGANQZiKVPAQtNKAKJ0eoMHQIyEXAEI8iMI8yZiOAQQVIZuWIwqciGNHNBNEIEPEQqAEALEEACAFI7hfAZAUAQePAaGbgYIZAEAOVAQ5gAovsRCN4ZkK4bYRRRuZAchQUEAKKFQWuI8KaTouAK4BAWGVkRUZuOosABoZo0QUkGAC4RCNkYQZAbeOePo7QI0bAQEGYGAK4GgF6FEUQOADYnebY-AI0K4dkNQAARQYAgHgAuK2MKNxiwDqP7B7hUGaMDVgFriVAMGQGN2PneJwEIHrlYjqPlCGCICkGaPsGeLgBWOKLKJQDoDEBVXKSaMKKWG4BWLenWKQE2MIEKMIDMJ6KwAoXJLuLgEeORNeLJJAD7kBOYGBKQFxUKIamCXZGaEEGJJAELBVWaI0JOBoE7yZL6IkAoRbwiFgCuG3xSJTGUED3IDUClIkgkiAA)

Reply:
Thanks for the tip! I used the prettier output.